src/pydocstyle/cli.py

```diff
@@ -53,7 +53,7 @@ def run_pydocstyle(use_pep257=False):
 
     count = 0
     for error in errors:
-        sys.stderr.write('%s\n' % error)
+        sys.stdout.write('%s\n' % error)
```

Comment:
Instead of checking that the count number is somewhere in the output, check that the it's the last line of `stdout`.